alkira/resource_alkira_service_infoblox.go

```diff
@@ -0,0 +1,373 @@
+package alkira
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAlkiraInfoblox() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manage Infoblox services",
+		Create:      resourceInfoblox,
+		Read:        resourceInfobloxRead,
+		Update:      resourceInfobloxUpdate,
+		Delete:      resourceInfobloxDelete,
+
+		Schema: map[string]*schema.Schema{
+			"anycast": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Description: "Defines the AnyCast policy to be used with the Infoblox Service. " +
+					"Based on this AnyCast policy some implicit route policies and prefix lists get " +
+					"generated. These route policies and prefix lists will have the prefix " +
+					"ALK-SYSTEM-GENERATED-INFOBLOX. These route policies and prefix lists cannot " +
+					"be deleted or modified directly their lifecycle is bound by the Infoblox " +
+					"services that are configured on the network. AnyCast may be enabled/disabled " +
+					"at the instance level as well. For AnyCast to be enabled for an instance it " +
+					"MUST be enabled both at the service and the instance level. If AnyCast is " +
+					"NOT enabled at the service level it will stay disabled for all instances.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Description: "Defines if AnyCast should be enabled. Default is `false`",
+							Type:        schema.TypeBool,
+							Required:    true,
```

Comment:
If default is `false`, we could make it optional with a default value as `false`.